### DISCUSSION TITLE
Instagram session file?

### DISCUSSION THREAD

**Discussion by andyfcx** [bellingcat/auto-archiver#122] [Q&A]
Hi, I am trying instagram_archiver. 
In the config file, there are 3 fields need to specified, I got username and password,
but I have no idea how can I get a `session_file`, please help.

Thanks.

**Comment by msramalho** [comment]
Hi @andyfcx ,

If you don't have a session file yet, one will be created by trying to login with the provided username/password, so choose the name you want or leave it empty (the default value is `secrets/instaloader.session`). 

You can see the code logic [here](https://github.com/bellingcat/auto-archiver/blob/590d3fe824a57b520396d27f59d4a90f275d06ce/src/auto_archiver/archivers/instagram_archiver.py#L29-L41).

**Comment by msramalho** [comment]
Please see this https://github.com/bellingcat/auto-archiver/issues/231#issuecomment-2709083605

but echoing the message from that comment:


![Image](https://github.com/user-attachments/assets/ad50368a-3e34-4527-9e92-cc5ffc4a6ecd)
